### ISSUE TITLE
Cannot receive tokens from testnet

### ISSUE THREAD

**Issue by zaesero** [CovenantSQL/CovenantSQL#84]
Followed the [quickstart](https://testnet.covenantsql.io/quickstart ) generated wallet and verified my 
twitter post but cannot get the tokens, the website is stuck on the last step. Can anyone help get the test tokens?

**Comment by auxten** [comment]
log shows:
```
covenantsql_faucet      | time="2018-10-11T08:51:24Z" level=error msg="open session to 0000011839f464418166658ef6dec09ea68da1619a7a9e0f247f16e0d6c6504d from pool failed: session shutdown" caller="pool.go:132 rpc.(*SessionPool).Get"
covenantsql_faucet      | time="2018-10-11T08:52:02Z" level=info msg="ReadFull failed: read tcp 172.19.0.3:39076->120.79.254.36:11105: read: connection reset by peer" caller="conn.go:70 crypto/etls.(*CryptoConn).Read"
covenantsql_faucet      | time="2018-10-11T08:52:02Z" level=warning msg="allocate nonce for transaction failed: unexpected EOF" caller="verifier.go:233 main.(*Verifier).dispenseOne"
```

follow up later

**Comment by leventeliu** [comment]
Seems the session pool ran into an unrecoverable state:

1. Get a session from the pool: succeed.
2. Open net.Conn on the cached session: failed, session shutdown.
3. Dial new connection to node and hand over to `yamux` as new session: succeed.
4. Open net.Conn on the new session: succeed.
5. Read from the new connection: failed, reset by peer.
6. Application layer retry, go to 1.

Restarted faucet service to fix currently, definitely needs to follow up.

**Comment by auxten** [comment]
Fixed in #98 add an test case here https://github.com/CovenantSQL/CovenantSQL/blob/develop/rpc/rpcutil_test.go#L277
